src/lib/dataEntities/networks.ts

```diff
@@ -514,6 +516,25 @@ export function mapL2NetworkToArbitrumNetwork(
   }
 }
 
+/**
+ * Asserts that the given object has a token bridge. This is useful because not all Arbitrum network
+ * operations require a token bridge.
+ *
+ * @param network {@link ArbitrumNetwork} object
+ * @throws ArbSdkError if the object does not have a token bridge
+ */
+export function assertHasTokenBridge<T extends ArbitrumNetwork>(
```

Comment:
tiny rename + let's export this at index?

```suggestion
export function assertArbitrumNetworkHasTokenBridge<T extends ArbitrumNetwork>(
```